src/test/java/de/tum/in/www1/artemis/localvcci/LocalVCLocalCIIntegrationTest.java

```diff
@@ -518,8 +519,8 @@ void testFetchPush_assignmentRepository_examMode() throws Exception {
         // Working time
         exam.setStartDate(ZonedDateTime.now().minusHours(1));
         examRepository.save(exam);
-        studentExam.setExam(exam);
-        studentExamRepository.save(studentExam);
+        // studentExam.setExam(exam);
+        // studentExamRepository.save(studentExam);
```

Comment:
I think you should remove these lines, or is there a reason why they are still here?